Cartfile

```diff
@@ -1,6 +1,6 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
 binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 6.1.1
-github "mapbox/MapboxDirections.swift" ~> 0.27.3
+github "mapbox/MapboxDirections.swift" "maxspeedAnnotations"
```

Comment:
This PR depends on mapbox/MapboxDirections.swift#367. Once that PR lands, we’ll need to change this dependency to `master`. Finally, once we publish a new release of MapboxDirections.swift, we’d change this line and corresponding lines in the podspecs to require the new release.